doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -128,9 +138,10 @@ Key: "CHASSIS_MODULE|DPU0"
 #### DPU shutdown sequence
 * There could be two possible sources for DPU shutdown. 1. A configuration change to DPU "admin_status: down" 2. The GNOI logic can trigger it.
 * The GNOI server runs on the DPU even after the DPU is shutdown.
```

Comment:
@rameshraghupathy if the DPU is shut how can GNOI server run?

Reply:
> @rameshraghupathy if the DPU is shut how can GNOI server run?

Meant to say pre-shutdown.
The GNOI server runs on the DPU even after the DPU is pre-shutdown and listens until the graceful shutdown finishes.
Fixed
@prgeor